### PR TITLE
Add Metal announcement

### PR DESCRIPTION
**Pull request by wipfli**
@dnomadb here you can make a pull request now to this branch since it is in the main repo...

### CODE REVIEW

**Comment by wipfli** [comment]
image

**Comment by wipfli** [comment]
This is ready for review...

**Comment by wipfli** [comment]
Associated tweet: https://github.com/maplibre/maplibre/discussions/105#discussioncomment-3997922

File: content/news/2022-10-28-metal-announcement/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+MapLibre GL Native is planning for modularization and migration to Metal, Apple's graphics framework, for rendering on iOS mobile devices.
+
+<p>
+<img src="metal.png" alt="Metal" style="max-width:600px;width:100%">
+</p>
+
+We are happy to announce that Stamen Design, with technical leadership by Wet Dog Weather and funding from AWS, have begun planning for the modularization of MapLibre rendering and a separate proposal for migrating the iOS SDK from OpenGL to Metal.
+
+Stamen provides full-stack cartographic services to clients who rely heavily on projects like MapLibre, which tie all the parts of the modern mapping stack – tiles, styles, and SDKs – together. Steve Gifford from Wet Dog Weather will be the technical lead for this project, and Stephanie May, Damon Burgett, and Camilla Mahon from Stamen will support the effort to get these plans in front of the community.
```
Comment (nyurik) [inline content/news/2022-10-28-metal-announcement/index.md:17]: I would move info about Stamen to the end. First would be best to describe the *what* is being done, what plans we have, the roadmap, etc.

Comment (maxammann) [reply content/news/2022-10-28-metal-announcement/index.md:17]: Im also missing some more technical information. Except if this should only be a short announcement.

**Comment by wipfli** [comment]
I would avoid using the term "Metal support" because we used this already in MapLibre GL Native for the MetalANGLE implementation. We named the branch for this the Metal support branch: https://github.com/maplibre/maplibre-gl-native/tree/metal-support

**Comment by wipfli** [comment]
I reordered a bit. The intention is indeed to not be too technical in this announcement, @maxammann, but I think we can have technical blog posts here in the future...

**Review by lseelenbinder — APPROVED** [review]

**Pull request merged**